pkg/agent/clean/orphan_bindings.go

```diff
@@ -108,6 +116,7 @@ func newOrphanBindingsCleanup(clientConfig *restclient.Config) (*orphanBindingsC
 	bc := orphanBindingsCleanup{
 		namespaces:   k8score.Core().V1().Namespace(),
 		prtbs:        rancherManagement.Management().V3().ProjectRoleTemplateBinding(),
+		prtbCache:    wContext.Mgmt.ProjectRoleTemplateBinding().Cache(),
```

Comment:
It would be simpler to follow the convention of the file and continue using the norman controllers instead of trying to create and use a wrangler context for this one thing. We can switchover the whole file in one go later.

Reply:
Agreed, but there was a problem with the Norman cache. It wasn't getting populated. I wonder if this has to do with how management is set up. Maybe it should be instantiated with `NewFactoryFromConfigWithOptions`, rather than just `NewFactoryFromConfig`? That's why I just switched to Wrangler controller's cache.

Reply:
The controllers might never be getting started. The rancher clustermanager starts the controllers on startup, if you're just trying to use the cache directly without starting the controllers then they won't get populated.

Reply:
Moved from the indexer solution to just a PRTB list and then processing them, since a controller would initially do the same anyways, and we don't need a controller for a short-lived process.